src/server.ts

```diff
@@ -388,6 +404,55 @@ server.tool(
   },
 );
 
+server.tool(
+  "rerun-workflow",
+  "Rerun a workflow component thread in a session",
```

Comment:
```suggestion
  "Start a new workflow session by copying an existing sessions values",
```
But the point is that you should try and avoid the special verbs in the descriptions and expand them out. It helps with clarity.